src/components/BrewerySearch.js

```diff
@@ -15,21 +17,28 @@ export default class BrewerySearch extends Component {
 
   onSearchClick = e => {
     const { search } = this.state
-    const { handleSearch } = this.props
-    if (search) {
-      handleSearch(search)
+    let searchTrim = search.trim();
+    if (searchTrim.length > 1) {
+      this.props.handleSearch(searchTrim)
+    }else{
+      this.setState({ alertMessage:'true'});
+      setTimeout(
+        function() {
+          this.setState({ alertMessage:'false'});
+        }
+        .bind(this),
+        3000
+      );
```

Comment:
1) Use an arrow-function instead of trying to bind this.setState to an anonymous function.
2) I think 3000ms is too long, especially as there's no option to dismiss the Alert until setTimeout completes.